content/blog/posts/2022-o3de-user-survey-report.md

```diff
@@ -0,0 +1,49 @@
+
+The survey collected user feedback on the O3DE user experience based on nine common game development workflows, from Onboarding, Game Design, Actor Development, World Building, Look Development, Multiplayer, to Team Collaboration, Engine Extension, and Packaging & Deployment. Within each workflow, users were asked to rate:
+
+- Workflow satisfaction: how satisfied they are with the workflow
+- Workflow confidence: how confident they feel they achieved their stated objectives after completing the workflow
+- Workflow efficiency: how efficient they feel their time spent on the workflow
+- Feature satisfaction: how satisfied they are using the features within the workflow.
+
+#### What Did We Find?
+
+Results from the [final survey report](https://github.com/o3de/sig-ui-ux/blob/main/user-research/2022%20Q2%20O3DE%20User%20Experience%20Survey.md) suggest areas of improvement across all nine workflows surveyed. The two highest performing workflows are Onboarding and Look Development, which suggests the data-driven plan to focus on improvements for these two workflows has been received positively, enabling users to finish these workflows within a reasonable amount of time. Meanwhile, six workflows, including Actor Development, World Building, Multiplayer, Team Collaboration, Engine Extension, Packaging and Deployment, received lower ratings, which suggests these workflows require the greatest improvement.
+
+{{< image-width src="/images/blog/user-survey-2022/survey-2022.png" width="100%" alt="Mean Ratings Across 3 User Experience Metreics" >}}
+
+We also asked users to rate their experience with the features within the workflow. Based on the data, the Help System across O3DE, the integration with the version control tools, the toolings (e.g. Terrain and Dynamic Vegetation) in the World Building workflow, the Physics System, and the asset management experience (including Asset Browser) received lower satisfaction ratings (ranging from 2.5 to 2.8). The UX performance of these features could be the reason that drove lower ratings in their designated workflows.
+
+#### Using Qualitative Data to Deepen Understanding
+
+While this quantitative survey data can point to areas of overall improvement in O3DE’s user experience, we need qualitative data to inform a deeper understanding of the results to guide the future evolution of the engine’s user experience. During the summer months, qualitative interviews were conducted with 20 community members, focused on the five common workflows users employ when building their content with O3DE: Onboarding Experience, Actor Development, Look Development, Multiplayer, and Engine Extension.
```

Comment:
Since the words are so similar, italicizing can make sure people don't miss it.
```suggestion
While this _quantitative_ survey data can point to areas of overall improvement in O3DE’s user experience, we need _qualitative_ data to inform a deeper understanding of the results to guide the future evolution of the engine’s user experience. During the summer months, qualitative interviews were conducted with 20 community members, focused on the five common workflows that users employ when building their content with O3DE: Onboarding Experience, Actor Development, Look Development, Multiplayer, and Engine Extension.
```